README.md

```diff
@@ -4,20 +4,55 @@
+<br />
+
+## About the NPM package
+
+The NPM package for Office.js is a copy of what gets published to the official "evergreen" Office.js CDN, at **<https://appsforoffice.microsoft.com/lib/1/hosted/office.js>**. 
+
+While the Office.js CDN contains all currently available Office.js APIs at any moment in time, each version of the NPM package for Office.js contains only the Office.js APIs that were available at the point in time when that version of the NPM package was created. 
+
+### Target scenarios
+
+The NPM package for Office.js is intended as a way for you to obtain your own (non-CDN) copy of the Office.js files, which you can then statically serve from your own site instead of using the CDN. This NPM package is primarily provided to address the following scenarios:
+
+1. If you are developing an add-in behind a firewall, where accessing the Office.js CDN is not possible.
+
+2. If you need offline access to the Office.js APIs (for example, to facilitate offline debugging).
+
+Additionally, using the alpha and beta versions of the NPM package can facilitate faster-cadence beta-testing (relative to the slower-cadence beta CDN at **<https://appsforoffice.microsoft.com/lib/beta/hosted/office.js>**).
+
+### Best practices
+
+Best practices for using the Office.js NPM package include:
+
+- Refresh your NPM package periodically (to gain access to new APIs and/or bug fixes that may not have been available in your current version of the package).
+
+- Use the NPM package according to the instructions in [Using the NPM package](#using-the-npm-package); do not try to import the NPM package as you might commonly do with other NPM packages.
+
+- Do not use the NPM package in an add-in that you submit for publication to [AppSource](https://appsource.microsoft.com/marketplace/apps?product=office). Add-ins that are published to AppSource must use the Office.js CDN.
 
 <br />
 
-## Installing
+## Installing the NPM package
 
-To install "office-js" locally via the NPM package, run
+To install "office-js" locally via the NPM package, run the following command:
 
     npm install @microsoft/office-js --save
 
-Once installed, the Office.js script reference can be used as
+<br />
+
+## Using the NPM package
+
+Installing the NPM package locally creates a set of static Office.js files in the `node_modules\@microsoft\office-js\dist` folder of the directory where you ran the `npm install` command. To use the NPM package, do the following:
+
+1. Copy the contents of this folder as-is to the destination of your choosing.
+
+2. Reference that location in a `<script>` tag within the HTML file in your add-in project.
 
-    <script src="node_modules\@microsoft\office-js\dist\office.js"></script>
+For example, if you add the contents of the `dist` folder to the `assets` directory of your web server (`https://mysite.com`), then you'd add the following `<script>` tag to your HTML file:
 
+    <script src="https://mysite.com/assets/office.js"></script>
```

Comment:
It's better to use a relative path that's relative off of the root of the url.  That way it works on localhost too
E.g.: `/assets/office-js/office.js`